examples/history-read/history-read.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2018-2019 opcua authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/gopcua/opcua"
+	"github.com/gopcua/opcua/debug"
+	"github.com/gopcua/opcua/id"
+	"github.com/gopcua/opcua/ua"
+)
+
+func main() {
+	endpoint := flag.String("endpoint", "opc.tcp://localhost:4840", "OPC UA Endpoint URL")
+	nodeID := flag.String("node", "", "NodeID to read")
+	flag.BoolVar(&debug.Enable, "debug", false, "enable debug logging")
+	flag.Parse()
+	log.SetFlags(0)
+
+	c := opcua.NewClient(*endpoint)
+	if err := c.Connect(); err != nil {
+		log.Fatal(err)
+	}
+	defer c.Close()
+
+	node, err := ua.ParseNodeID(*nodeID)
```

Comment:
I've named this `id` in the other examples since this is just the id of the node.